src/ShortidType.php

```diff
@@ -31,35 +36,47 @@ public function getSQLDeclaration(array $column, AbstractPlatform $platform): st
         return $platform->getVarcharTypeDeclarationSQL($field).' '.$platform->getColumnCollationDeclarationSQL('utf8_bin');
     }
 
-    public function convertToPHPValue($value, AbstractPlatform $platform)
+    /**
+     * {@inheritdoc}
+     */
+    public function convertToPHPValue($value, AbstractPlatform $platform): ?Shortid
     {
         if (empty($value)) {
-            return;
+            return null;
         }
-        if (ShortId::isValid($value)) {
+        if (ShortId::isValid((string) $value)) {
```

Comment:
unneeded without strict types

Reply:
Again like above: I'll revert that in dissent.